CHANGELOG.md

```
@@ -1,3 +1,602 @@
+# [4.0.0-alpha.1](https://github.com/ar-io/ar-io-sdk/compare/v3.24.0-alpha.1...v4.0.0-alpha.1) (2026-06-05)
+
+
+* feat!: port Solana backend from solana-ar-io monorepo ([6613766](https://github.com/ar-io/ar-io-sdk/commit/66137664a5fd136740308a166a293f2e9b499435)), closes [#5](https://github.com/ar-io/ar-io-sdk/issues/5)
+
+
+### Bug Fixes
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)